dvc/config.py

```diff
@@ -246,6 +248,9 @@ def __init__(
         else:
             self.dvc_dir = os.path.abspath(os.path.realpath(dvc_dir))
 
+        self.wtree = WorkingTree(self.dvc_dir)
+        self.tree = tree.tree if tree else self.wtree
```

Comment:
Having to `tree.tree` because of `Repo.tree`'s setter :slightly_frowning_face: Looks like that will need to be revisited as well.